dipy/workflows/tracking.py

```diff
@@ -48,14 +48,17 @@ def _get_direction_getter(self, strategy_name):
         elif strategy_name.lower() in ["closestpeaks", "cp"]:
             msg = "ClosestPeaks"
             direction_getter = ClosestPeakDirectionGetter
+        elif strategy_name.lower() in ["eudx", ""] and 'pam' in kwargs.keys():
```

Comment:
I do not think `eudx` will be selected as default since `deterministic` is the default value on `run` function. But, I agree to remove it and avoid confusion